src/ssl.c

```diff
@@ -985,8 +985,10 @@ int wolfSSL_SetTmpDH(WOLFSSL* ssl, const unsigned char* p, int pSz,
     if (pSz < ssl->options.minDhKeySz)
         return DH_KEY_SIZE_E;
 
+    #ifndef WOLFSSL_WPAS
```

Comment:
Why would a TLS client need to load DH parameters?  Ownership logic of DH parameters assumes explicitly loaded ones are sever side currently.